extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -76,8 +76,8 @@ export class AmpAd extends AMP.BaseElement {
 
       // TODO(tdrl): Check amp-ad registry to see if they have this already.
       if (!a4aRegistry[type] ||
-          !a4aRegistry[type](this.win, this.element) ||
-          this.win.document.querySelector('meta[name=amp-3p-iframe-src]')) {
+          this.win.document.querySelector('meta[name=amp-3p-iframe-src]') ||
```

Comment:
I put it this way for minuscule performance reason where presumably the check for type existence in the config object is much less expensive than querying the DOM for remote.html meta existence.